examples/battery_storage_parallel.jl

```diff
@@ -0,0 +1,114 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# Solving an decision hazard battery storage problem using multiprocessed
```

Comment:
This is a lot better :+1: 

You still need to precise your notation in your description add the notation in parenthesis.